docs/OAuth2.md

```diff
@@ -40,6 +40,8 @@ Example of an authorization URL:
 https://linear.app/oauth/authorize?client_id=client1&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Foauth%2Fcallback&response_type=code&scope=read,write
 ```
 
+> Note: If the user has already authorized your application for the given scopes, they will be automatically redirected to your redirect url rather than having to approve your application again.
```

Comment:
```suggestion
If the user has already authorized your application for the given scopes, they will be automatically redirected to your redirect url rather than having to approve your application again.
```